multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -1,22 +1,25 @@
 # Unique name of Kubernetes cluster. In order to deploy
 # more than one cluster into the same AWS account, this
 # name must not conflict with an existing cluster.
-# clusterName: kubernetes
+
+clusterName: {{.ClusterName}}
+
+# DNS name routable to the Kubernetes controller nodes
+# from worker nodes and external clients. The deployer
+# is responsible for making this name routable
+externalDNSName: {{.ExternalDNSName}}
+
+### DO NOT CHANGE PARAMETERS ABOVE HERE ###
```

Comment:
nit: maybe move them below a "Do not change below". Little weird to read about configurables, then see a comment saying I shouldn't change them